index.html

```diff
@@ -206,8 +206,54 @@ <h2>Skills</h2>
           </div>
         </section>
          </div> 
-     <!-- </section> -->
-
+         <section class="contact-section">
+            <header class="contact-header">
+                <h2>
+                    I'm always interested in hearing about new projects,
+                     so if you'd like to chat please get in touch.
+                </h2>
+            </header>
+            <div class="contact-form">
+                <form action="https://formspree.io/f/mvolbpag" method="post">
+                    <input type="text" name="fullname" id="fullname" maxlength="30" placeholder="Full name">
+                    <input type="email" name="email" id="email" placeholder="Email address">
```

Comment:
- [ ] Please add the required attribute to the email input field. This will ensure that a value exists before submitting the form.